.github/workflows/lint-prose.yml

```diff
@@ -16,6 +16,7 @@ on:
 jobs:
   prose:
     runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.changed_files < 301 }}
```

Comment:
Why do we want to limit this action? Is it because of saving the number of action executions? Or does it take too long to run this job with bigger amount of changed files?

Reply:
We want to limit because Vale uses Reviewdog in a version that can't handle more than 300 changed filed ([issue](https://github.com/reviewdog/reviewdog/issues/1696)).

Reply:
Ok, I get it know. Thx.

Reply:
Could we add a comment or something? Without context, it would be hard to understand why we're doing this.

Reply:
Do we have any examples of this? How did you find out?

Reply:
I found out because #4401 is failing so I investigated and came across multiple people reporting this. I will add a comment